web-react-ts/src/pages/campaigns/components/buttons/DefaulterDetailsCard.tsx

```diff
@@ -56,27 +56,43 @@ function DefaulterDetailsCard(props: DefaulterDetailsCardProps) {
           </div>
         )}
 
-        <div className="text-success">
+        <div
+          className={church?.fellowshipEquipmentFilledCount === 0 ? '' : 'good'}
+        >
           Fellowship Filled Count: {church?.fellowshipEquipmentFilledCount}
         </div>
         {props.route === 'constituency/fellowship' ? (
-          <div className="text-success">
+          <div
+            className={church.equipmentRecord.pulpits === null ? 'bad' : 'good'}
+          >
             Constituency Filled:{' '}
             {church.equipmentRecord.pulpits === null ? 'No' : 'Yes'}
           </div>
         ) : (
-          <div className="text-success">
+          <div
+            className={
+              church?.constituencyEquipmentFilledCount === 0 ? '' : 'good'
```

Comment:
This condition should be checking whether constituencies fill is equal to the Total number of constituencies